Tasks/SshV0/task.json

```diff
@@ -104,6 +104,19 @@
             "defaultValue": true,
             "helpMarkDown": "If this option is selected, the build will fail when the remote commands or script write to STDERR.",
             "groupName": "advanced"
+        },
+        {
+            "name": "handshakeTimeout",
```

Comment:
For consistency, can we rename this `readyTimeout`? That way it corresponds directly to the ssh2 module

Reply:
Sure, I'll do that :)